deals-add.js

```diff
@@ -0,0 +1,223 @@
+var script = document.createElement('script');
+script.type = 'text/javascript';
+script.src = 'https://s.cdpn.io/3/clamp.js';
+document.head.appendChild(script);
+
+var script = document.createElement('script');
+script.type = 'text/javascript';
+script.src = 'https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment-with-locales.js';
+document.head.appendChild(script);
+
+var dealsContainer = document.getElementsByClassName("deals-container")[0];
+var currentCategory = 'Daily';
+var offset = 0;
```

Comment:
Will need to _namespace_ all of these properly too... Or wrap all of your functions in a javascript class.